public/app/plugins/datasource/prometheus/result_transformer.ts

```diff
@@ -74,6 +74,7 @@ export class ResultTransformer {
       datapoints: dps,
       query: options.query,
       target: metricLabel,
+      labels: metricData.metric,
```

Comment:
should probably be called tags, as that is what grafana internals calls series labels. 
https://github.com/grafana/grafana/blob/master/pkg/tsdb/models.go#L39

datasource abstraction also has two hooks named getTagKeys & getTagValues 

https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/elasticsearch/datasource.ts#L408